todo/src/reducers/index.js

```diff
@@ -0,0 +1,57 @@
+
+import { TO_DO_CREATE, TO_DO_DESTROY, TO_DO_COMPLETE, TO_DO_CLEAR } from '../actions';
+
+const initialState = {
+  toDos: []
+};
+
+export default (state = initialState, action) => {
```

Comment:
```suggestion
export default (state = [], action) => {
```

Reply:
Doesn't require to be inside of an object. Could even be a single number if we needed it to